desktop/core/src/desktop/templates/taskserver_list_tasks.mako

```diff
@@ -0,0 +1,40 @@
+<%!
```

Comment:
Is this only reachable under a new separate URL?

If it's accessible from the Hue UI you shouldn't need a separate new mako file

Reply:
Yes, this only reachable under a seperate URL. Created this file similar to "desktop/core/src/desktop/templates/threads.mako"